scripts/start.sh

```diff
@@ -7,8 +7,40 @@ set -e
 ScriptDir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 cd "$ScriptDir"
 
+# get the port from the REACT_APP_BACKEND_URI env variable
+envContent=$(grep -v '^#' ../webapp/.env | xargs)
+backendPort=$(echo $envContent | sed -n 's/.*:\([0-9]*\).*/\1/p')
+
 # Start backend (in background)
 ./start-backend.sh &
 
-# Start frontend
-./start-frontend.sh
+maxRetries=5
+retryCount=0
+retryWait=5
```

Comment:
nit: add the following comment like in the other script
```suggestion
retryWait=5  # set the number of seconds to wait before retrying
```